recap/converters/avro.py

```diff
@@ -147,7 +148,7 @@ def _from_recap(
                 # meaning from other types.
                 avro_schema.pop("aliases", None)
             case _:
-                raise ValueError(f"Unsupported Recap type: {recap_type}")
+                avro_schema["type"] = "unknown"
```

Comment:
I think we should continue to raise value error here.. "unkonwn" is not a valid Avro schema type. Parsers would fail with this.